0-bootstrap/modules/jenkins/variables.tf

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  Required variables
+*******************************************/
+
+variable "org_id" {
+  description = "GCP Organization ID"
+  type        = string
+}
+
+variable "billing_account" {
+  description = "The ID of the billing account to associate projects with."
+  type        = string
+}
+
+variable "default_region" {
+  description = "Default region to create resources where applicable."
+  type        = string
+  default     = "europe-west2"
```

Comment:
we should try and use a consistent default region. I believe the bootstrap module uses us-central1 https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/variables.tf#L44

Reply:
Done

Reply:
Done